File: sandbox/exercism/c/2020/gigasecond.c
/* Gigasecond from https://exercism.io */

#include "gigasecond.h"
#include <math.h>

time_t gigasecond_after(time_t time)
{
  time_t gigasecond = pow(10, 9);

  return gigasecond + time;
}

File: sandbox/exercism/go/2025/gigasecond.go
// https://exercism.org/tracks/go/exercises/gigasecond

package gigasecond

import "time"

func AddGigasecond(t time.Time) time.Time {
	gigasecond := time.Duration(1_000_000_000)
	return t.Add(time.Second * gigasecond)
}
